src/main/java/com/hyperwallet/clientsdk/util/HyperwalletEncryption.java

```diff
@@ -1,36 +1,18 @@
 package com.hyperwallet.clientsdk.util;
 
 import com.hyperwallet.clientsdk.HyperwalletException;
-import com.nimbusds.jose.Algorithm;
-import com.nimbusds.jose.EncryptionMethod;
-import com.nimbusds.jose.JOSEException;
-import com.nimbusds.jose.JWEAlgorithm;
-import com.nimbusds.jose.JWEHeader;
-import com.nimbusds.jose.JWEObject;
-import com.nimbusds.jose.JWSAlgorithm;
-import com.nimbusds.jose.JWSHeader;
-import com.nimbusds.jose.JWSObject;
-import com.nimbusds.jose.JWSSigner;
-import com.nimbusds.jose.Payload;
-import com.nimbusds.jose.crypto.RSADecrypter;
-import com.nimbusds.jose.crypto.RSAEncrypter;
-import com.nimbusds.jose.crypto.RSASSASigner;
-import com.nimbusds.jose.crypto.RSASSAVerifier;
-import com.nimbusds.jose.jwk.JWK;
-import com.nimbusds.jose.jwk.JWKSet;
-import com.nimbusds.jose.jwk.RSAKey;
+import com.nimbusds.jose.*;
```

Comment:
be careful with this type of import. It might import a lot of necessary classes. 
Also and most importantly IMO It becomes more straightforward to whom is reading de code to understand where does the Payload class comes from. In this case it comes from `com.nimbusds.jose.Payload`

Reply:
Hmm I think IntelliJ arranged it this way when I did optimized imports lol, I'll try changing them back